api/server/v1/server.proto

```diff
@@ -181,12 +181,42 @@ message Settings {
   uint32 default_role_id = 18;
 }
 
+// Settings represents a stripped-down version of PMM Server settings that can be accessed by users of all roles.
+message ReadOnlySettings {
+  // True if updates are enabled.
+  bool updates_enabled = 1;
+  // True if telemetry is enabled.
+  bool telemetry_enabled = 2;
+  MetricsResolutions metrics_resolutions = 3;
+  google.protobuf.Duration data_retention = 4;
+  // True if Advisor is enabled.
+  bool advisor_enabled = 7;
+  // Percona Platform user's email, if this PMM instance is linked to the Platform.
+  string platform_email = 8;
+  // True if Alerting is enabled.
+  bool alerting_enabled = 10;
+  // PMM Server public address.
+  string pmm_public_address = 11;
+  // Intervals between Advisor runs.
+  AdvisorRunIntervals advisor_run_intervals = 12;
```

Comment:
do we really need these settings to be public?

Reply:
Not really - they are shown on the advanced settings page, but non-admin roles can't access that page anyway - we can reserve the number as in the other cases